ts/src/pro/binance.ts

```diff
@@ -2018,8 +2019,10 @@ export default class binance extends binanceRest {
         }
         params = this.extend (params, { 'type': type, 'symbol': symbol }); // needed inside authenticate for isolated margin
         await this.authenticate (params);
+        let marginMode = undefined;
+        [ marginMode, params ] = this.handleMarginModeAndParams ('watchOrders', params);
         let urlType = type;
-        if (type === 'margin') {
+        if ((type === 'margin') || ((type === 'spot') && (marginMode !== undefined))) {
```

Comment:
Not really because according to a comment in this method spot and spot-margin use the same stream, but if a user wanted to use the unified approach for setting the marginMode I added handleMarginModeAndParams to omit the marginMode if it's set

Reply:
I'm okay with just closing this pull request if you want

Reply:
@Dan-krm we can keep it just for the marginMode omitting